### ISSUE TITLE
Database Setup Error

### ISSUE THREAD

**Issue by Justinzobel** [maxpozdeev/mytinytodo#9]
After selecting SQLite in the database setup screen I got:
```
myTinyTodo 1.6.4 Setup


Fatal Error: 'Call to a member function ex() on int' in /var/www/todo/setup.php:165

#0 {main}
```

**Comment by Justinzobel** [comment]
After removing the db file, adding it again (touch dbname.file and chown to www-data) it worked.

**Comment by maxpozdeev** [comment]
Can confirm this happens in PHP 8. Will be fixed.

**Comment by maxpozdeev** [comment]
Fixed in v1.6.5
